src/Tribe/Main.php

```diff
@@ -1594,9 +1594,7 @@ public function saved_organizers_dropdown( $current = null, $name = 'organizer[O
 		 */
 		public function admin_body_class( $classes ) {
 			global $current_screen;
-			if ( isset( $current_screen->post_type ) &&
-				 ( $current_screen->post_type == self::POSTTYPE || $current_screen->id == 'settings_page_tribe-settings' )
-			) {
+			if ( isset( $current_screen->post_type ) && ( $current_screen->post_type == self::POSTTYPE || $current_screen->id == 'settings_page_tribe-settings' ) ) {
```

Comment:
Collapsing this to a single line makes it harder to read, so I'm not a huge fan of this particular change.  If anything, _more_ spacing may help:

```php
if (
  isset( $current_screen->post_type )
  && (
    self::POSTTYPE === $current_screen->post_type
    || 'settings_page_tribe-settings' === $current_screen->id
  )
) {
```